start-up/moderation-tutorial.md

```diff
@@ -1,54 +1,55 @@
 # Moderation Tutorial
 
-## Intro
+This guide will give you the basic information needed to moderate your Discord server with MoonlightBot. It includes a list of commands and how they work, complete with their options, infractions, and log information to help you better understand how to use them
 
-This guide will give you the basic information needed to moderate your Discord server with MoonlightBot. It includes a list of commands and how they work, complete with their arguments, infractions, and log information to help you better understand how to use them
+## Command Options
 
-## Arguments
+These are options you may use while executing moderation commands. You can find a more in-depth guide about options in the [arguments page](/start-up/arguments.md)
 
-Arguments are options you may use while executing moderation commands. You can find a more in-depth guide about arguments in the [arguments page](/start-up/arguments.md)
+![A list of example options in a mute command](/.gitbook/assets/MuteArguments.png "Options Example")
 
-![](/.gitbook/assets/MuteArguments.png)
-
-- **`user`**: The target member of the command
-- **`reason`**: The reason for which you are executing the command
-- **`notify`**: Whether or not the bot will message the user after the command is executed
-- **`no-infraction`**: Whether or not the command will count towards the user's infractions
+- `user`: The user to execute the moderation action on
+- `reason`: The reason for which you are taking the action, useful for record keeping. The reason shows up in infractions, logs, and notifications from the bot
+- `notify`: Whether or not the bot will message the user after the command is executed, as shown below
+![An example of a notification from the bot](/.gitbook/assets/NotifyExample.png "Notify Example")
+- `no-infraction`: Whether or not the command will count towards the user's infractions
 
 {% hint style="info" %}
-If both the `notify` and `no-infraction` arguments are true, the target will not be notified in order to prevent harrassment via MoonlightBot
+If both the `notify` and `no-infraction` options are true, the target will not be notified in order to prevent harassment via MoonlightBot
```

Comment:
```suggestion
If both the `notify` and `no-infraction` options are true, the target will **not** be notified in order to prevent harassment via MoonlightBot
```